### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by yetieaterxb1**
- added cassandra seeding and querying
- added csv for seeding script(will use this for postgres)

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const cors = require('cors');
 const db = require('./db/index.js');
 require('dotenv').config();
 
-const port = process.env.PORT;
+const port = 3004;
```
Comment (mchung1128) [inline server/app.js:6]: Is there a reason you removed your environment variable here?

File: cassSeedAlsoViewed.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+const endCategoryId = 10000000;
+const getCategoryId = () => faker.random.number({ min: startCategoryId, max: endCategoryId });
+
+// generate seed data with ids provided otherwise default from 101 to 200 to given
+async function seeding() {
+  // ViewedItem.sync({ force: true });
+  console.time('SeedingTime');
+  const seedAlsoViewedItems = [];
+  const startId = 101;
+  const endId = 20002000;
```
Comment (mchung1128) [inline cassSeedAlsoViewed.js:66]: Where are these start and end ID numbers being pulled from? Are the random?

**Review by mchung1128 — COMMENTED** [review]
Looks pretty good.

File: cassSeedAlsoViewed.js (hunk 2 above)
Comment (yetieaterxb1) [reply cassSeedAlsoViewed.js:66]: I was using random numbers to tests seeding from the legacy code.

File: server/app.js (hunk 1 above)
Comment (yetieaterxb1) [reply server/app.js:6]: I had some issues and will refactor with it.

**Pull request merged**